cl_sii/rtc/data_models_aec.py

```diff
@@ -648,6 +653,49 @@ def slug(self) -> str:
     # Custom Methods
     ###########################################################################
 
+    def verify_signature(
+        self,
+        xml_doc: xml_utils.XmlElement,
+    ) -> Optional[bool]:
+        """
+        Verify signature of AEC XML document ``xml_doc``.
+        :param xml_doc: An XML document, as returned by ``xml_utils.parse_untrusted_xml()``.
+        :raises Exception: on unrecoverable errors
+        """
+        signature_verified: Optional[bool]
+
+        if self.signature_x509_cert_der is None:
+            raise ValueError("Field 'signature_x509_cert_der' can not be None.")
+
+        signature_x509_cert: crypto_utils.X509Cert = crypto_utils.load_der_x509_cert(
```

Comment:
How are you handling the `ValueError` that is raised by `crypto_utils.load_der_x509_cert` when the certificate is not loadable (see https://github.com/fyntex/lib-cl-sii-python/pull/261)?

Or is it intentional because you want the `ValueError` to be propagated to the caller? If that's not your intention, then you should wrap the call to `crypto_utils.load_der_x509_cert` in a try-except and set `signature_verified` appropriately (`None`?).

Reply:
Good catch, I should have considered this case. I think the best solution is to return `None` instead of propagating the error.

Reply:
> I think the best solution is to return `None` instead of propagating the error.

I agree.